src/gui/gui.cc

```diff
@@ -1060,6 +1062,10 @@ void PCSX::GUI::endFrame() {
                         _("Assembly"));
     }
 
+    if (m_disassembly.m_show) {
```

Comment:
Re-add a check here for the CPU type, in case we eventually add the boolean to the configuration file.

Reply:
Let me know if you don't like the && , and I will change it to a nested if-statement

Reply:
&& is perfectly fine.